sql/telemetry/desktop_savant_events_v1/view.sql

```diff
@@ -0,0 +1,187 @@
+          WHEN event_name = 'SAVANT - install addon finish' THEN 'addon'
+          WHEN event_name = 'SAVANT - remove addon start' THEN 'addon'
+          WHEN event_name = 'SAVANT - remove addon finish' THEN 'addon'
+          WHEN event_name = 'SAVANT - load login form' THEN 'loginform'
+          WHEN event_name = 'SAVANT - submit login form' THEN 'loginform'
+          WHEN event_name = 'SAVANT - use pw manager' THEN 'pwmgr'
+          WHEN event_name = 'SAVANT - asked add pw' THEN 'pwmgr'
+          WHEN event_name = 'SAVANT - saved pw' THEN 'pwmgr'
+          WHEN event_name = 'SAVANT - updated pw' THEN 'pwmgr'
+          END AS value
+      UNION ALL SELECT 'entrypoint' AS key, CASE
+          WHEN event_name = 'SAVANT - made search' THEN event_object
+          END AS value
+      UNION ALL SELECT 'type' AS key, CASE
+          WHEN event_name = 'SAVANT - made search' THEN event_value
+          END AS value
+      UNION ALL SELECT 'engine' AS key, CASE
+          WHEN event_name = 'SAVANT - made search' THEN `moz-fx-data-derived-datasets.udf.get_key`(event_map_values, 'engine')
+          END AS value
+      UNION ALL SELECT 'addonid' AS key, CASE
+          WHEN event_name = 'SAVANT - enable addon' THEN event_value
+          WHEN event_name = 'SAVANT - disable addon' THEN event_value
+          WHEN event_name = 'SAVANT - install addon start' THEN event_value
+          WHEN event_name = 'SAVANT - install addon finish' THEN event_value
+          WHEN event_name = 'SAVANT - remove addon start' THEN event_value
+          WHEN event_name = 'SAVANT - remove addon finish' THEN event_value
+          END AS value
+      UNION ALL SELECT 'can-record-submit' AS key, CASE
+          WHEN event_name = 'SAVANT - load login form' THEN `moz-fx-data-derived-datasets.udf.get_key`(event_map_values, 'canRecordSubmit')
+          END AS value
+      UNION ALL SELECT 'flow-id' AS key, CASE
+          WHEN event_name = 'SAVANT - load login form' THEN `moz-fx-data-derived-datasets.udf.get_key`(event_map_values, 'flow_id')
+          WHEN event_name = 'SAVANT - submit login form' THEN `moz-fx-data-derived-datasets.udf.get_key`(event_map_values, 'flow_id')
+          WHEN event_name = 'SAVANT - use pw manager' THEN `moz-fx-data-derived-datasets.udf.get_key`(event_map_values, 'flow_id')
+          WHEN event_name = 'SAVANT - asked add pw' THEN `moz-fx-data-derived-datasets.udf.get_key`(event_map_values, 'flow_id')
+          WHEN event_name = 'SAVANT - saved pw' THEN `moz-fx-data-derived-datasets.udf.get_key`(event_map_values, 'flow_id')
+          WHEN event_name = 'SAVANT - updated pw' THEN `moz-fx-data-derived-datasets.udf.get_key`(event_map_values, 'flow_id')
+          END AS value
+      UNION ALL SELECT 'add-type' AS key, CASE
+          WHEN event_name = 'SAVANT - asked add pw' THEN event_value
+          END AS value
+  ) WHERE VALUE IS NOT NULL) AS event_props_2,
+  ARRAY<STRING>[] AS user_props
```

Comment:
We're missing user_properties that are [hardcoded](https://github.com/mozilla/telemetry-streaming/blob/5844343d7a2049ba230c288a4c7cee85cfd7612e/src/main/scala/com/mozilla/telemetry/pings/MobileEvent.scala#L41-L59) per ping.